### DISCUSSION TITLE
Macro environment properties from XML

### DISCUSSION THREAD

**Discussion by daniele-baccega** [FLAMEGPU/FLAMEGPU2#1069] [Q&A] [answered]
Hi,
is it possible to initialize a macro environment property from the XML file? I need to initialize a tensor with three dimensions, but I didn't understand how to do it from the XML file.

Thank you!

**Comment by Robadob** [comment, marked as answer]
Not currently.

This feels like an oversight on our part.

I've created an [issue](https://github.com/FLAMEGPU/FLAMEGPU2/issues/1070), which I may be able to handle in July when I'm back on 50% time improving FLAMEGPU.
